library/std/src/os/net/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! Linux and Android-specific definitions for socket options.
+
+#![unstable(feature = "tcp_quickack", issue = "96256")]
+#![doc(cfg(any(target_os = "linux", target_os = "android",)))]
+
+pub mod net;
```

Comment:
This is `os::net::net`? That leads to confusing re-exports. We need better naming here.

Reply:
agreed, changed to `os::net::tcp`